Added Yammer Files API (#839)
MicrosoftDocs/OfficeDocs-O365ITPro: yammer/manage-security-and-compliance/export-yammer-enterprise-data.md

To set up automatic recurring exports, use the Yammer API. For information about the Data Export API, go to [/export](https://go.microsoft.com/fwlink/?LinkID=534735) on the Yammer Developer Center. 

<a name="ExportFiles"></a>
## Export Yammer Files via API

> The Yammer Files Export API lets Verified Administrators archive and export files in Yammer storage in an asynchronous manner. This API is intended to be used when there is a significant volume of files to be exported from Yammer. Please see the documentation at https://aka.ms/YammerFilesAPI 

Reviewer: ```suggestion
The Yammer Files Export API lets verified administrators archive and export files in Yammer storage in an asynchronous manner. This API is intended to be used when there is a significant volume of files to be exported from Yammer. For more information, please see [Yammer Files Export API](https://aka.ms/YammerFilesAPI).
```
